unitTests/ui/DebugForResultTest.ts

```diff
@@ -32,13 +32,11 @@ export function DebugForResultTest() {
       Expression: "@spacekey=SitecoreV3" Score: 0
       Expression: "@syssource=answers" Score: 0
       Expression: "@syssource==(apices70,apices65,"Web Scraper - apisitecore30")" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/en/cloud/Search.htm"" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/en/ces/7.0/Search.htm"" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/en/ces/6.5/Search.htm"" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/fr/cloud/Search.htm"" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/fr/ces/7.0/Search.htm"" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/fr/ces/6.5/Search.htm"" Score: 0
-      Expression: "@sysuri=="https://onlinehelp.coveo.com/en/cloud/404_error.htm"" Score: 0
+      Expression: "@sysuri=="https://docs.coveo.com/en/1547/"" Score: 0
```

Comment:
Mmmm... I think you should revert those changes; I wouldn't touch UTs.

@samisayegh , @ThibodeauJF what do you think?

Reply:
@dmbrooke, did you make these changes to (A) fix the test/stop it failing? Or (B) just to adjust the urls to look like the new format?

If (A), I am curious to know why the test was failing.
If (B), then these changes are okay.

Reply:
I already said this in chat but I'll repeat here to just so everyone is on the same page:

I made the changes to (B) adjust the urls to look like the new format. A few of the links pointed to dead pages so I thought it would be fine to remove them